apps/cross-components/src/index.ts

```diff
@@ -0,0 +1,15 @@
+export { test } from 'lib/internal';
```

Comment:
This duplicates the registered task but is a lot easier to bundle it for web like that, need to refactor this.